### ISSUE TITLE
Icon did not update in Windows search bar

### ISSUE THREAD

**Issue by Tekbr** [Microsoft/vscode#33685]
- VSCode Version: 1.16.0 - insiders (cc2cedf  - Date 2017-09-01T12:34:53.752Z)
- OS Version: Windows 10

-- Sorry for the English, I used Google Translator --

When using the Windows 10 Search Bar and search by Visual Studio Code.

![vs](https://user-images.githubusercontent.com/19866231/29973832-4099213a-8f07-11e7-832c-6b85af5b6b1c.PNG)

Reproduces without extensions: Yes

**Comment by DickvdBrink** [comment]
Probably just some Windows cache I think, because it is working for me

**Comment by Tekbr** [comment]
-- Sorry for the English, I used Google Translator --

This is very strange because it is the first program I see not updating. I see no reason to have to reinstall and install again to resolve this issue.

@DickvdBrink Did you ever do a clean install?

**Comment by rianadon** [comment]
The same thing happened to me, but after a while the icon changed. As @DickvdBrink said it's probably a Windows cache.

**Comment by chrisdias** [comment]
This is most likely a problem with Windows caching icons. if it still exists, please reopen.

**Comment by Tekbr** [comment]
-- Sorry for the English, I used Google Translator --

@chrisdias, I did some more testing on a VM ...

Problem continues. but I did some testing deleting the `IconCache.db` file. I reinitialized Windows.

In fact, the icons will update except the shortcut on the desktop.

Test by installing:
**VSCode 1.16.1
VSCode - Insiders (old version, less than 1.15.0)
Firefox beta 56.0b12**

Put the taskbar and desktop icon and then install the **Firefox beta 57.0b10, VSCode 1.17.2 and VSCode - Insiders 1.18.0**

It might not update (I did two tests and once Firefox updated and the other did not, with new VMs for each test). Delete the `IconCache.db` file. Restart Windows.

Note that the only desktop icon you updated was Firefox. (at least for me - I can try to make a video / gif).


In fact this case is about Windows search. But there is this case (#36282) about the icon in the work area that was closed by @joaomoreno. Should we reopen that case, or focus on that case here? (Generalizing the title of the question).

Thoughts.
